api/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.ccci.gto</groupId>
     <artifactId>global-registry-client</artifactId>
-    <version>0.2.2-SNAPSHOT</version>
+    <version>0.2.3-SNAPSHOT</version>
```

Comment:
shouldn't need to bump the snapshot version, after we merge I'll push a release build